core/src/main/scala/filodb.core/query/RangeVector.scala

```diff
@@ -324,6 +324,7 @@ final class SerializedRangeVector(val key: RangeVectorKey,
                                   override val outputRange: Option[RvRange]) extends RangeVector with
                                           SerializableRangeVector with java.io.Serializable {
 
+  def numBytes(): Int = containers.map(_.numBytes).sum
```

Comment:
Can you please confirm that this does not alter state of record builders, and this is an O(1) call within each RecordContainer ? Looks like this is being called while the result is being built. Want to be sure.